packages/lodestar/src/constants/network.ts

```diff
@@ -16,6 +16,11 @@ export enum Method {
   BeaconBlocksByRoot = "beacon_blocks_by_root",
 }
 
+// Methods that returns 1 single response_chunk
+export const SINGLE_CHUNK_METHODS = [Method.Status, Method.Ping, Method.Metadata];
+// Request only method
+export const NO_CHUNK_METHODS = [Method.Goodbye];
```

Comment:
I don't feel like those arrays are best way forward, as it's quite easy to forget to add it.
We could modify Method enum to contain more informations.
Like:
```javascript
export const Methods = {
   Method.Status: {
      responseType: MethodResponseType.SingleRespone
   },
   Method.Ping: {
      responseType: MethodResponseType.NoResponse
   },
   Method.BlocksByRange: {
      responseType: MethodResponseType.Stream
   }
}
```
Maybe ti makes more sense to add more stuff later like `maxRequestPerSecond` etc

Reply:
this could also hold ssz types for request and response so we can remove those stuff from encoders